src/client/components/pages/RoomSchedule/RoomSchedule.tsx

```diff
@@ -222,28 +231,28 @@ const RoomSchedule: FunctionComponent = () => {
                 options={semesterOptions}
                 onChange={updateTerm}
               />
-              <Combobox
-                options={fullRoomList
-                  .map((room): { value: string; label: string; } => ({
-                    label: room.name,
-                    value: room.id,
-                  }))}
-                currentValue={null}
-                label="Select a room"
-                isLabelVisible={false}
-                placeholder="Select a room"
-                onOptionSelected={({
-                  selectedItem: {
-                    value: id, label: displayName,
-                  },
-                }) => {
-                  setCurrentRoom({ id, displayName });
-                }}
-                filterFunction={(option, inputValue) => {
-                  const re = new RegExp(inputValue, 'i');
-                  return re.test(option.label);
-                }}
-              />
+              <RoomSelectorWrapper>
+                <Combobox
+                  options={fullRoomList
+                    .map((room): { value: string; label: string; } => ({
+                      label: room.name,
+                      value: room.id,
+                    }))}
+                  currentValue={null}
+                  label="Select a room"
```

Comment:
Thanks for this suggestion. I applied some placeholder text to the dropdown. I left out the word "room" since the label next to the combobox already references "room".